kai/constants.py

```diff
@@ -32,3 +33,22 @@
 
 PATH_TESTS = os.path.join(PATH_GIT_ROOT, "tests")
 PATH_TEST_DATA = pathlib.Path(os.path.join(PATH_GIT_ROOT, "tests/test_data"))
+
+
+def __clean_env() -> dict[str, Any]:
+    """
+    Returns the environment that should be used when calling out to a subprocess
+    """
+    env: dict[str, Any] = dict(os.environ)  # make a copy of the environment
+    lp_key = "LD_LIBRARY_PATH"  # for GNU/Linux and *BSD.
```

Comment:
Should this only be run if we are in the pyinstaller-created binary?

Reply:
Do you anticipate that this would cause a problem if not running in pyinstaller?

Reply:
In a sufficiently strange environment, probably

Reply:
locally it works fine for me outside of the binary, I think we can leave it for now and revisit if someone reports an issue.